locksmith/__tests__/controllers/v2/customEmailController.test.ts

```diff
@@ -16,6 +16,10 @@ const customEmailContent = `Custom Email Content`
 
 vi.mock('../../../src/operations/wedlocksOperations', () => {
   return {
+    EmailTemplate: vi.fn(),
```

Comment:
I don't understand why this is needed? The sendEmail function takes the template.

Reply:
Please respond @SVell ?